recipe/meta.yaml

```diff
@@ -1,15 +1,16 @@
 {% set name = "synphot" %}
-{% set version = "0.3" %}
+{% set version = "1.0.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 05855e2b9c64a545ed1d14d9d11e9d6c88cf071886bb9406d02699023ccc1971
+  sha256: 5d09a56d301b3ae78244cd22c407509b78fab84addfe80e6f662bebd99c95940
 
 build:
+  detect_binary_files_with_prefix: false
```

Comment:
This didn't help either. Should I remove it? Not sure what it does.

Reply:
Not sure...and now that astropy job fails with a `pkg_resources` error, so 🤷‍♂️.

Reply:
I think @saimn fixed it on `master`, so that should go away in RC2 if that patch is backported.

Reply:
Hmm, I can't find that PR I thought existed on astropy but I found https://stackoverflow.com/questions/7446187/no-module-named-pkg-resources